packages/gasket-plugin-https/lib/index.js

```diff
@@ -156,6 +156,52 @@ module.exports = {
           link: 'README.md#servers',
           parent: 'start',
           after: 'terminus'
+        }],
+        configurations: [{
+          name: 'http',
+          link: 'README.md#configuration',
+          description: 'HTTP port',
+          type: 'number'
```

Comment:
This can be either a port number or a config object
```suggestion
          description: 'HTTP port or config object',
          type: 'number | object'
```